std/net/curl.d

```diff
@@ -2803,14 +2847,14 @@ struct HTTP
         // Force post if necessary
         if (p.method != Method.put &&
             p.method != Method.post &&
-            p.method != Method.options)
+            p.method != Method.options &&
+            p.method != Method.patch)
             p.method = Method.post;
 
-        if (p.method == Method.put)
-            lenOpt = CurlOption.infilesize_large;
-        else
-            // post
+        if (p.method == Method.post)
             lenOpt = CurlOption.postfieldsize_large;
+        else
+            lenOpt = CurlOption.infilesize_large;
```

Comment:
Which is the difference between infilesize_large and postfieldsize_large? On other language bindings I see they use post data for patch method.